src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java

```diff
@@ -192,14 +192,14 @@ public void unfollowIfDiscussionIsFollowed(int threadNumber) {
     WebElement followButton = wait.forElementVisible(followButtonBy);
 
     if (followButton.getText().contains("Following")) {
-      PageObjectLogging.log("unfollowIfDiscussionIsFollowed",
+      Log.log("unfollowIfDiscussionIsFollowed",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)